sdk/keyvault/keyvault-admin/package.json

```diff
@@ -104,10 +104,10 @@
   "sideEffects": false,
   "dependencies": {
     "@azure/abort-controller": "^1.0.0",
-    "@azure/core-http": "^1.2.0",
-    "@azure/core-lro": "^1.0.2",
+    "@azure/core-http": "^2.0.0",
```

Comment:
While this branch is _only_ to release keyvault-keys, because of the way keyvault-common is used across packages it was easier to just update all 4 packages